internal/driver/interactive.go

```diff
@@ -82,9 +95,15 @@ func interactive(p *profile.Profile, o *plugin.Options) error {
 					continue
 				}
 				// Allow group=variable syntax by converting into variable="".
-				if v := pprofVariables[value]; v != nil && v.group == name {
-					if err := pprofVariables.set(value, ""); err != nil {
-						o.UI.PrintErr(err)
+				if okValues := groups[name]; okValues != nil {
```

Comment:
Why is this check required?

Reply:
Or why it's not inside the else branch below?

Reply:
Done.